tools/legal-review/engine/org.eclipse.jgit.org.eclipse.jgit-6.3.0.202209071007-r/copyright-keep

```diff
@@ -0,0 +1,342 @@
+Copyright (C) 2018-2021, Andre Bossert <andre.bossert@siemens.com>
+Copyright (C) 2019 Google LLC and others
+Copyright (C) 2018-2022, Andre Bossert <andre.bossert@siemens.com> and others
+Copyright (C) 2019, Google Inc. and others
+Copyright (C) 2019, Google LLC and others
+Copyright (C) 2018-2022, Andre Bossert <andre.bossert@siemens.com>
+Copyright (C) 2019, Google LLC. and others
+Copyright (C) 2019, Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2020, Google LLC and others
+Copyright (C) 2019, Marc Strapetz <marc.strapetz@syntevo.com>
+Copyright (C) 2019, Matthias Sohn <matthias.sohn@sap.com> and others
+Copyright (C) 2019, Tim Neumann <tim.neumann@advantest.com>
+Copyright (C) 2020 Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2020, Google LLC. and others
+Copyright (C) 2020, Matthias Sohn <matthias.sohn@sap.com> and
+Copyright (C) 2020, Michael Dardis. and others
+Copyright (C) 2020, Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2021 Simeon Andreev <simeon.danailov.andreev@gmail.com> and others
+Copyright (C) 2021 Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2021 Thomas Wolf <thomas.wolf@paranor.ch> and others.
+Copyright (C) 2021, 2022 Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2021, Fabio Ponciroli <ponch@gerritforge.com>
+Copyright (C) 2021, Google Inc. and others
+Copyright (C) 2021, Google LLC. and others
+Copyright (C) 2021, Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2022 Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (C) 2022,  Matthias Sohn <matthias.sohn@sap.com> and others
+Copyright (C) 2022, Fabio Ponciroli <ponch78@gmail.com> and others
+Copyright (C) 2022, Google Inc. and others
+Copyright (C) 2022, Simeon Andreev and others
+Copyright (C) 2022, Tencent.
+Copyright (C) 2022, Workday Inc.
+Copyright (c) 2014 Konrad Kügler and others
+Copyright (c) 2019 Matthias Sohn <matthias.sohn@sap.com>
+Copyright (c) 2019, Google LLC  and others
+Copyright (c) 2019, Thomas Wolf <thomas.wolf@paranor.ch> and others
+Copyright (c) 2020 Thomas Wolf <thomas.wolf@paranor.ch>
+Copyright (c) 2020, 2022 Julian Ruppel <julian.ruppel@sap.com> and others
+Copyright (c) 2020, Google LLC  and others
+Copyright (c) 2020, Google LLC and others
+Copyright (c) 2021 Qualcomm Innovation Center, Inc.
+Copyright 2017 Marc Stevens <marc@marc-stevens.nl>, Dan Shumow <danshu@microsoft.com>
+Redistributions in binary form must reproduce the above copyright notice,
+Redistributions of source code must retain the above copyright
+Redistributions of source code must retain the above copyright notice, this
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+and other copyright owners as documented in the project's IP log.
+copyright notice, this list of conditions and the following
+other copyright owners as documented in the project's IP log.
```

Comment:
These also look suspicious, please double-check them as the ones above.

Reply:
fixed